src/utilcode/staticnohost/CMakeLists.txt

```diff
@@ -8,5 +8,9 @@ endif(WIN32)
 add_library_clr(utilcodestaticnohost STATIC ${UTILCODE_STATICNOHOST_SOURCES})
 
 if(CLR_CMAKE_PLATFORM_UNIX)
-  target_link_libraries(utilcodestaticnohost  nativeresourcestring)
+  if(CLR_CMAKE_PLATFORM_DARWIN)
+    target_link_libraries(utilcodestaticnohost  nativeresourcestring  coreclrpal)
```

Comment:
The target_link_libraries can be used multiple times in an additive manner. Could you please keep the original as common and add just the `target_link_libraries(coreclrpal) conditionally?

Reply:
Sure, will do.